frontend/src/utils/link.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react'
+
+export function SecureLink(href: string, children: string) {
+    return (<React.Fragment>
```

Comment:
No need for `(` or `Fragment`, just return the `<a>` directly. Also should `children` be of type `ReactNode`?

Reply:
Getting an error about scope without the fragment @bcherny ...

Reply:
What's the error?